### PR TITLE
Feature/cheyenne gnu

### PR DESCRIPTION
**Pull request by natalie-perlin**
Changes for Cheyenne to build with modules restored/saved in module collection, to prevent over-loading system-installed modules with earlier versions

### CODE REVIEW

**Comment by kgerheiser** [comment]
Are you merging into the correct branch? Do you want to merge into develop?

File: config/config_cheyenne_gnu.sh [hunk 1]
```diff
@@ -2,34 +2,34 @@
...
 export   MAKE_CHECK=N
 export MAKE_VERBOSE=N
 export   MAKE_CLEAN=N
 export DOWNLOAD_ONLY=N
 export STACK_EXIT_ON_FAIL=Y
-export WGET="wget -nv"
+export WGET="wget -nv --no-check-certificate"
 
 module purge
-module unuse /glade/u/apps/ch/modulefiles/default/compilers
```
Comment (climbfuji) [inline config/config_cheyenne_gnu.sh:22]: Have you tried this with netcdf 4.8.1? My guess is that you will run into trouble, but hopefully I am wrong.

Comment (natalie-perlin) [reply config/config_cheyenne_gnu.sh:22]: Yes, this build was made with netcdf version_c 4.8.1 and version_f 4.5.4

**Comment by natalie-perlin** [comment]
@kgerheiser - a branch feature/mac probably needs merging with develop first! And then feature/cheyenne could be checked on merging conflict. This may or may not needs to be merged with the develop branch, as it extensively uses functionality of Lmod to save/restore module collections.

**Comment by natalie-perlin** [comment]
Will close this branch, and further do pull requests into existing feature/cheyenne branch

**Pull request closed without merging**